packages/@okta/vuepress-site/docs/guides/authenticators-okta-email/main/aspnet/integratechallengemagiclink.md

```diff
@@ -309,7 +230,7 @@ Build a form that allows the user to enter the One-Time Password (OTP) sent to t
 </section>
 ```
 
-### 9. User clicks the email magic link
+### 7. User clicks the email magic link
```

Comment:
The diagram lists this as "7. Click the email magic link". Make this text consistent with the diagram.